agent/pkg/up9/main.go

```diff
@@ -145,6 +150,30 @@ func SyncEntries(syncEntriesConfig *shared.SyncEntriesConfig) error {
 	return nil
 }
 
+func upsertModel(token string, model string, envPrefix string) error {
+	upsertModelUrl, _ := url.Parse(fmt.Sprintf("https://trcc.%s/models/%s", envPrefix, model))
+
+	authHeader := getAuthHeader(false)
+	req := &http.Request{
+		Method: http.MethodPost,
+		URL:    upsertModelUrl,
+		Header: map[string][]string{
+			authHeader: {token},
+		},
+	}
+
+	response, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return fmt.Errorf("failed request to upsert model, err: %v", err)
+	}
+
+	if response.StatusCode != 201 && response.StatusCode != 409 {
```

Comment:
maybe add comment to explain it is ok if we created it or it is already exists (which happens with 409)